WordPress/src/main/java/org/wordpress/android/ui/prefs/MyProfileActivity.java

```diff
@@ -174,4 +175,17 @@ public void onEventMainThread(PrefsEvents.MyProfileDetailsChanged event) {
             refreshDetails();
         }
     }
+
+    public void onEventMainThread(PrefsEvents.MyProfileDataLoadSaveError event) {
+        if (!isFinishing()) {
+            if (event != null
+                    && event.mVolleyError != null
+                    && event.mVolleyError.networkResponse != null
+                    && event.mVolleyError.networkResponse.statusCode == 401) {
```

Comment:
You can simplify this using [VolleyUtils.statusCodeFromVolleyError()](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/util/VolleyUtils.java#L34).